README.md

```diff
@@ -42,39 +42,39 @@ When you finish the instructions below, you will have the beginning of a best pr
 
 ### GitLab
 
-1. On a new server, install GitLab
+1. On a new server, install GitLab CE.
  - https://about.gitlab.com/downloads/
 
-2. After GitLab is installed you may sign into the web UI with the `root` user and password `5iveL!fe`
+2. After GitLab is installed, sign into the web UI with the user `root`
 
-2. In the GitLab UI, create a group called `puppet` ( this is case sensitive )
+2. In the GitLab UI, create a group called `puppet`
  - http://doc.gitlab.com/ce/workflow/groups.html
 
-3. In the GitLab UI, make a user for yourself
+3. In the GitLab UI, make yourself a user to edit and push code.
 
-4. From your laptop or development machine, make an SSH key to link with your user.
- - Note: this is used for your laptop to communicate with your GitLab server to push code
+4. From your laptop or development machine, make an SSH key and link it with your GitLab user.
+ - Note: The SSH key allows your laptop to communicate with the GitLab server and push code.
  - https://help.github.com/articles/generating-ssh-keys/
  - http://doc.gitlab.com/ce/ssh/README.html
 
-7. In the GitLab UI, add your user to the `puppet` group
- - Make sure to give your user at least master permissions so you can complete the below steps
+7. In the GitLab UI, add your user to the `puppet` group.
+ - You must give your user at least master permissions to complete the following steps.
  - Read more about permissions:
     - https://gitlab.com/gitlab-org/gitlab-ce/blob/master/doc/permissions/permissions.md
 
-8. In the GitLab UI, create a project called `control-repo` and set its Namespace to the `puppet` group
+8. In the GitLab UI, create a project called `control-repo` and set its Namespace to the `puppet` group.
 
-10. On your laptop, clone this PuppetLabs-RampUpProgram control repo
+10. On your laptop, clone this PuppetLabs-RampUpProgram control repo.
```

Comment:
You could choose to split setting up GitLab server from cloning the repo after this step. No big deal right now, but easier later on if/when you add other Git hosting services.